include/aktualizr-lite/api.h

```diff
@@ -99,12 +99,18 @@ class DownloadResult {
   };
   Status status;
   std::string description;
-  std::function<InstallResult()> Install;
 };
 
 std::ostream &operator<<(std::ostream &os, const InstallResult &res);
 std::ostream &operator<<(std::ostream &os, const DownloadResult &res);
 
+class InstallContext {
```

Comment:
At first glance, I don't get a need in the `InstallContext`. The previous version worked for primary while "download" and "install" for secondaries are supposed to be done by a client out of the libaklite scope.